### PR TITLE
add example workflows

### PR DESCRIPTION
**Pull request by AtomicFS**
Add couple of basic example workflows, which act as documentation and as tests.

When executed, they actually exposed some problems with path handling, which should be fixed now.

I will add later one job which will use coreboot with either linux or edk2 as payload, but at the moment there is no code in coreboot building to actually handle this.

### CODE REVIEW

File: dist/invoke-action.mjs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const VERSION = '0.0.1';
+
+main();
+
+function main() {
+  console.log("Node Version: ", process.version);
+
+  // Compile Go program
+  console.log("Building ...");
+  const goBuildReturns = spawnSync('go', ['build', '-o', '../action.bin'], {
```
[the remark this answers is not in the document]
Comment (AtomicFS) [reply dist/invoke-action.mjs:14]: That was my first attempt, so technically yes. But then you have problems with paths.

The working directory should be `repo` and not `repo/action`.

File: action.yml [hunk 2]
```diff
@@ -143,4 +146,4 @@ inputs:
 
 runs:
   using: 'node16'
```
Comment (AtomicFS) [inline action.yml:148]: btw, shouldn't this be like `node20`?

Comment (MDr164) [reply action.yml:148]: If things work with the newer version feel free to bump. I just pinned it previously to not deal with GitHub upgrading what ubuntu-latest has as a default.

File: dist/invoke-action.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (MDr164) [reply dist/invoke-action.mjs:14]: Ah right, we would have to deal with the work dir. Let's discuss that tomorrow then.

**Review by MDr164 — APPROVED** [review]

**Pull request merged**